tests/integration/components/polaris-resource-list/filter-control/date-selector-test.js

```diff
@@ -0,0 +1,617 @@
+import { module, test, skip } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import {
+  render,
+  find,
+  findAll,
+  triggerEvent,
+  click,
+  blur,
+} from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { DateFilterOption } from '@smile-io/ember-polaris/components/polaris-resource-list/filter-control/date-selector';
+import DatePickerComponent from '@smile-io/ember-polaris/components/polaris-date-picker';
+
+const textFieldInputSelector = '.Polaris-TextField input';
+const selectedDaySelector = '.Polaris-DatePicker__Day--selected';
+
+const dateOptionType = {
+  past: [
+    DateFilterOption.PastWeek,
+    DateFilterOption.PastMonth,
+    DateFilterOption.PastQuarter,
+    DateFilterOption.PastYear,
+    DateFilterOption.OnOrBefore,
+    DateFilterOption.OnOrAfter,
+  ],
+  future: [
+    DateFilterOption.ComingWeek,
+    DateFilterOption.ComingMonth,
+    DateFilterOption.ComingQuarter,
+    DateFilterOption.ComingYear,
+    DateFilterOption.OnOrBefore,
+    DateFilterOption.OnOrAfter,
+  ],
+  full: [
+    DateFilterOption.PastWeek,
+    DateFilterOption.PastMonth,
+    DateFilterOption.PastQuarter,
+    DateFilterOption.PastYear,
+    DateFilterOption.ComingWeek,
+    DateFilterOption.ComingMonth,
+    DateFilterOption.ComingQuarter,
+    DateFilterOption.ComingYear,
+    DateFilterOption.OnOrBefore,
+    DateFilterOption.OnOrAfter,
+  ],
+};
+
+function getOptionsValuesList(options) {
+  if (!options) {
+    return [];
+  }
+
+  return options.map((option) => {
+    return typeof option === 'string' ? option : option.value;
+  });
+}
+
+async function triggerChangeEventWithValue(selector, value) {
```

Comment:
not 💯 but you might be able to just use [`fillIn`](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#fillin) instead of this custom method

Reply:
I tried `fillIn` but couldn't get it to work... 🤷‍♂️